web_poet/_typing.py

```diff
@@ -18,7 +18,7 @@ def is_generic_alias(obj) -> bool:
 
 
 def get_generic_parameter(cls):
-    for base in cls.__orig_bases__:
+    for base in getattr(cls, "__orig_bases__", []):
```

Comment:
Have you encountered an issue with `cls.__orig__bases__`?

Reply:
Yes, it doesn't work on cases like:

```python
@handle_urls("example.com", to_return=Product)
class CustomProductPageDataTypeOnly(web_poet.Injectable):
    ...
```